modules/face/CMakeLists.txt

```diff
@@ -1,3 +1,20 @@
 set(the_description "Face recognition etc")
-ocv_define_module(face opencv_core opencv_imgproc opencv_objdetect WRAP python)
+set(__commit_hash "6c209354186c3711cb46e10dacd75f195896e4a6")
+set(__file_hash "0f1fd479dc93a1d4bce12ad4c2b68d46")
+ocv_download(
+    FILENAME face_landmarks_model.dat
+    HASH
```

Comment:
```
HASH ${__file_hash}
```

It is better to put this block after `ocv_define_module` statement.

Also please squash your commits into one to remove intermediate large changes from git history.